README.md

```diff
@@ -167,8 +172,7 @@ We have a number of environments that are split between CDC and non-CDC Azure En
 
 ##### Internal
 
-The Internal environment is meant to be the Wild West.  Meaning anyone can push to it to test something, and there is no
-requirement that only good builds be pushed to it.  Use the Internal environment if you want to test something in a
+The Internal environment is designed to be the Wild West, meaning anyone can push changes without restrictions. It allows for testing various configurations without the requirement that only stable builds be pushed.  Use the Internal environment if you want to test something in a
```

Comment:
I reworded this a little to explain the colloquialism better for any who may not be familiar.

Reply:
Maybe add a note to ping the engineering channel if you are using the internal environment so that you aren't stepping over others?